src/lib/transition_frontier_controller_tests/test_ledger_catchup.ml

```diff
@@ -43,15 +45,23 @@ let%test_module "Ledger catchup" =
           let catchup_breadcrumbs_reader, catchup_breadcrumbs_writer =
             Strict_pipe.create Synchronous
           in
+          let unprocessed_transition_cache =
+            Transition_handler.Unprocessed_transition_cache.create ~logger
+          in
           let best_breadcrumb = Transition_frontier.best_tip peer.frontier in
           let best_transition =
             Transition_frontier.Breadcrumb.transition_with_hash best_breadcrumb
+            |> Transition_handler.Unprocessed_transition_cache.register
+                 unprocessed_transition_cache
+            |> Or_error.ok_exn
           in
           Strict_pipe.Writer.write catchup_job_writer best_transition ;
           Ledger_catchup.run ~logger ~network ~frontier:me
-            ~catchup_breadcrumbs_writer ~catchup_job_reader ;
+            ~catchup_breadcrumbs_writer ~catchup_job_reader
+            ~unprocessed_transition_cache ;
           let expected_breadcrumbs =
-            Transition_frontier.path_map peer.frontier best_breadcrumb ~f:Fn.id
+            Transition_frontier.path_map peer.frontier best_breadcrumb
+              ~f:Cache_lib.Cached.phantom
```

Comment:
is phantom just used to lift into the `Cached.t` context without actually caching something (for type-glueing or something)? If so, can we rename to `don't_store` or something like that (and also add some comments explaining what the phantom does)

Reply:
the name phantom confuses me because I expect it in the phantom type context

Reply:
well, it is basically a "value phantom" :)

I documented it in my latest commit (pushing soon), if it's still not clear there I can rename it. Don't like `don't_store` though as it changes the semantics (you don't store a `Cached.t`, a `Cached.t` is stored).

Reply:
Changed to `pure` as discussed.